.github/workflows/publish-to-pypi.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install setuptools wheel twine pytest
+        pip install pylint
```

Comment:
```suggestion
        pip install setuptools wheel twine
```

Reply:
Oh yes - nice spot! The error message that took me so long to track down was that `pylint` depends on `wrapt` which needs `setuptools` in order to install, so the installs do need to be this way round.